allauth/account/views.py

```diff
@@ -654,6 +654,12 @@ def logout(self):
                                   messages.SUCCESS,
                                   'account/messages/logged_out.txt')
         auth_logout(self.request)
+        signal_kwargs = {}
+        signals.user_logged_out.send(sender=self.request.user.__class__,
+                                    request=self.request,
+                                    user=self.request.user,
```

Comment:
Thank you for reply. Can't we send signal before 
`adapter.logout(self.request)` or put it into adapter before `django_logout(request)`?
In this case we still have user in request. The signal could be called 'pre_logout' then.